src/main/java/com/blamejared/crafttweaker/impl_native/entity/ExpandLivingEntity.java

```diff
@@ -251,4 +255,28 @@ public static boolean attackEntityAsMob(LivingEntity internal, Entity entityIn)
     public static boolean isWaterSensitive(LivingEntity internal) {
         return internal.isWaterSensitive();
     }
+    
+    @ZenCodeType.Method
+    public static void attackEntityFrom(LivingEntity internal, DamageSource damageSource, float amount) {
+        internal.attackEntityFrom(damageSource, amount);
+    }
+    
+    /**
+     * Gets the item stack of the entity in given slot.
+     * @param mutable if true, the method will return a {@link MCItemStackMutable}, instead of a copy one. Use it with caution.
+     * @return The item stack of the entity in given slot.
+     */
+    @ZenCodeType.Method
+    public static IItemStack getItemStackFromSlot(LivingEntity internal, EquipmentSlotType slot, @ZenCodeType.Optional boolean mutable) {
```

Comment:
Why the additional mutable parameter?
People could just do `getItemStackFromSlot(EquipmentSlot.HEAD).mutable()`

Reply:
https://github.com/CraftTweaker/CraftTweaker/blob/afc14e659e12351f295edfbe1535863b63395d09/src/main/java/com/blamejared/crafttweaker/impl/item/MCItemStack.java#L25

We copy the origin item stack in the constructor of MCItemStack. Changing a copied one is meaningless.

Reply:
> We copy the origin item stack in the constructor of MCItemStack. Changing a copied one is meaningless. (if you say to use setItemStackToSlot later, I won't say anything later)


Huh, when did that happen?
@jaredlll08 ?